lib/util/apply-disable-directives.js

```diff
@@ -32,26 +32,27 @@ function compareLocations(itemA, itemB) {
  * comment for two different rules is represented as two directives).
```

Comment:
The business logic changes (i.e. the changes that actually handle directive comments rather than just passing options through the API) are all in this file.